zebrad/src/components/mempool.rs

```diff
@@ -88,6 +90,28 @@ enum ActiveState {
     },
 }
 
+/// Mempool configuration section.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields, default)]
+pub struct Config {
+    /// The transaction cost limit
+    pub tx_cost_limit: u32,
+    /// Max amount of minutes for transactions to be in recently eviced
+    pub eviction_memory_minutes: u32,
+}
+
+// we like our default configs to be explicit
+#[allow(unknown_lints)]
+#[allow(clippy::derivable_impls)]
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            tx_cost_limit: 80000000,
```

Comment:
Readability Nitpick: Rust allows separators, so large numbers are easier to read.

```suggestion
            tx_cost_limit: 80_000_000,
```